corehq/apps/userreports/views.py

```diff
@@ -624,7 +624,8 @@ def page_context(self):
             'report_type': self.report_type,
             'form': report_form,
             'editing_existing_report': bool(self.existing_report),
-            'property_options': [p._asdict() for p in report_form.data_source_properties.values()],
+            'report_column_options': [p.to_dict() for p in report_form.report_column_options.values()],
+            'data_source_indicators': [p._asdict() for p in report_form.data_source_properties.values()],
```

Comment:
The first list contains `ColumnOption` objects (which define a `to_dict()` method), the second list contains `namedtuple` objects (which have a built in method `_asdict()`).